src/set_theory/surreal/basic.lean

```diff
@@ -290,6 +292,21 @@ noncomputable instance : linear_ordered_add_comm_group surreal :=
   decidable_le := classical.dec_rel _,
   ..surreal.ordered_add_comm_group }
 
+/-- Casts a `surreal` number into a `game`. -/
+def to_game : surreal →+o game :=
+{ to_fun := lift (λ x _, ⟦x⟧) (λ x y ox oy, quot.sound),
+  map_zero' := rfl,
+  map_add' := by { rintros ⟨_, _⟩ ⟨_, _⟩, refl },
+  monotone' := by { rintros ⟨_, _⟩ ⟨_, _⟩, exact id } }
+
+theorem zero_to_game : to_game 0 = 0 := rfl
+
+@[simp] theorem one_to_game : to_game 1 = 1 := rfl
+
+@[simp] theorem nat_to_game : ∀ n : ℕ, to_game n = n
```

Comment:
This is true for any `add_monoid_hom` that sends one to one, see https://leanprover-community.github.io/mathlib_docs/data/nat/cast.html#map_nat_cast'

Reply:
How do I express that `1` is sent to `1`?

Reply:
Otherwise LGTM.
bors d+